test/services/prompter/user-prompts.test.js

```diff
@@ -53,50 +55,59 @@ describe('services > prompter > user prompts', () => {
           expect.assertions(2);
           requests.push('email');
           const userPrompts = new UserPrompts(requests, envConfig, prompts, program);
-          expect(prompts).toHaveLength(0);
 
+          expect(prompts).toHaveLength(0);
           userPrompts.handleEmail();
-
           expect(prompts).toHaveLength(1);
+          resetParams();
         });
 
         it('should add a prompt with the correct configuration', () => {
           expect.assertions(4);
+          requests.push('email');
+          const userPrompts = new UserPrompts(requests, envConfig, prompts, program);
+          userPrompts.handleEmail();
```

Comment:
Test changed to remove SonarJS same function errors.
This changes also makes those tests less interdependent.